src/components/Router/index.js

```diff
@@ -70,38 +67,25 @@ class MainRouter extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      roomId: null,
-      socket: io(process.env.DESKTOP_SYNC_URL),
-      mobileConnected: false,
-      mobileInitialStep: null
+      mobileInitialStep: null,
     }
-
-    this.state.socket.on('joined', this.setRoomId)
-    this.state.socket.on('get config', this.sendConfig)
-    this.state.socket.emit('join', {})
-  }
-
-  setRoomId = (data) => {
-    this.setState({roomId: data.roomId})
   }
 
-  sendConfig = () => {
+  mobileConfig = () => {
     const {documentType, options} = this.props
     const {steps, token} = options
-    const config = {steps, token, documentType, step: this.state.mobileInitialStep}
-    this.state.socket.emit('message', {room: this.state.roomId, event: 'config', payload: config})
-    this.setState({mobileConnected: true})
+    return {steps, token, documentType, step: this.state.mobileInitialStep}
```

Comment:
why does the cross device mobile need to send so much stuff?

Reply:
This is sent from the desktop to the mobile. I think we need it all